airflow/providers/microsoft/mssql/hooks/mssql.py

```diff
@@ -51,6 +51,15 @@ def __init__(
         self.schema = kwargs.pop("schema", None)
         self._sqlalchemy_scheme = sqlalchemy_scheme
 
+    @staticmethod
+    def get_ui_field_behaviour() -> Dict[str, Any]:
+        """Returns custom UI field behaviour for MSSQL Connection."""
+        return {
+            "relabeling": {
+                "schema": "Database",
+            },
```

Comment:
Lets add this to base DB api hook instead -- it'll right for all the subclasses much more than calling it schema currently is!

Reply:
https://github.com/apache/airflow/pull/26732

Reply:
inheritance doesn't work for these methods though unfortunately.

aside... one thing that's unfortunate about relabeling is it doesn't help much when defining creds in env vars.  for that case it's a bit nicer to have them named properly as extras... though not much to do about that now i guess.